media/js/src/multipleChoiceQuestion.jsx

```diff
@@ -41,9 +41,7 @@ export const MultipleChoiceQuestion = ({
                 </header>
                 <div className="simulation__step-content">
                     <p>
-                        You have conpleted all the hypothesis tests in this
-                        section. Based on what you&rsquo;ve learned from this
-                        exercise, which of the following is true?
+                        {question}
```

Comment:
This change is already in the code. It's weird that the old code is still here. Can you try rebasing with the new main?

Reply:
Rebase done